### PR TITLE
Fix #2250: Change load error message

### LINKED ISSUE #2250: Change "Oops, something went wrong" page title.
Currently when the editor doesn't start we show this...

image

However, I think we should just say something like **Thimble could not start** so that it makes it more obvious to the user what's going on when they are reaching out to us via email.

### PR DESCRIPTION
**Pull request by MhdTlb**

### CODE REVIEW

**Comment by MhdTlb** [comment]
Fix for #2250

File: locales/en-US/messages.properties [hunk 1]
```diff
@@ -20,21 +20,22 @@ termsOfUse=Terms of Use
 privacyPolicy=Privacy Policy
 thimbleFooter=Thimble is part of the Mozilla Learning Network. <br/>Learn to teach like mozilla at <a href="http://learning.mozilla.org">learning.mozilla.org</a>.
 pageTitleGetInvolved=Thimble by Mozilla - Get Involved
+############
```
Comment (gideonthomas) [inline locales/en-US/messages.properties]: why was this change made?

File: locales/en-US/messages.properties [hunk 2]
```diff
@@ -20,21 +20,22 @@ termsOfUse=Terms of Use
...
-## Editor ##
-############
 
 # Loading the editor
 loadingProject=Loading Project
 loadingProjectSupportSuggestion=If the project doesn't load, <a title="Thimble help page" href="https://github.com/mozilla/thimble.mozilla.org/wiki/Project-not-Loading-Help">get help here</a>.
-somethingWentWrong=Thimble could not start.
+somethingWentWrong=Oops, something went wrong!
 reloadThimbleMessage=Please reload your browser to try loading Thimble again. <button class="refresh-browser">Reload Now</button>
+errorCouldNotStart=Thimble could not start.
```
Comment (gideonthomas) [inline locales/en-US/messages.properties]: This string should be "Oops, something went wrong!"

File: locales/en-US/messages.properties [hunk 3]
```diff
@@ -20,21 +20,22 @@ termsOfUse=Terms of Use
...
 
-############
-## Editor ##
-############
 
 # Loading the editor
 loadingProject=Loading Project
 loadingProjectSupportSuggestion=If the project doesn't load, <a title="Thimble help page" href="https://github.com/mozilla/thimble.mozilla.org/wiki/Project-not-Loading-Help">get help here</a>.
-somethingWentWrong=Thimble could not start.
+somethingWentWrong=Oops, something went wrong!
```
Comment (gideonthomas) [inline locales/en-US/messages.properties]: this key should not exist anymore

**Comment by gideonthomas** [comment]
@flukeout does the text look right?

**Pull request merged**